public/app/features/dashboard-scene/edit-pane/DashboardEditPaneRenderer.test.tsx

```diff
@@ -0,0 +1,72 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import { selectors } from '@grafana/e2e-selectors';
+import { setPluginImportUtils } from '@grafana/runtime';
+import { SceneGridLayout, SceneTimeRange, SceneVariableSet, VizPanel } from '@grafana/scenes';
+
+import { getPanelPlugin } from '../../../../../packages/grafana-data/test/helpers/pluginMocks';
```

Comment:
```suggestion
import { getPanelPlugin } from '@grafana/data/test';
```

I think we can use this instead of relative import